### DISCUSSION TITLE
ASUS ACPI and not installing the ASUS system control interface

### DISCUSSION THREAD

**Discussion by strawtaco** [seerge/g-helper#1118] [Q&A]
Regarding the issue of not being able to connect to ASUS ACPI and not installing the ASUS system control interface. But it still doesn't work after installation, even if it's restarted
Requesting your help

**Comment by seerge** [comment]
What kind of device are you using?

**Reply by strawtaco** [reply]
ROG strix G713PV

**Reply by seerge** [reply]
Just make sure to have https://dlcdnets.asus.com/pub/ASUS/nb/Image/CustomComponent/ASUSSystemControlInterfaceV3/ASUSSystemControlInterfaceV3.exe installed (most probably you already have it tho)

**Reply by strawtaco** [reply]
Yes, I have already installed it

**Reply by seerge** [reply]
Well, app clearly can't connect to it. Does armoury crate work ok on your device atm ?

Do you have any exotic / non standard windows or user account ?

**Reply by strawtaco** [reply]
The CPU temperature displayed on the Armory Crate is 0 and the fan speed cannot be adjusted in custom mode

**Comment by seerge** [comment]
@strawtaco well, then it's clearly something is broken in your system. I don't know what :) 

Try to check your system 
https://support.microsoft.com/en-au/topic/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system-files-79aa86cb-ca52-166a-92a3-966e85d4094e

Or even do a fresh install mb if you want.

**Reply by strawtaco** [reply]
I have reinstalled the system and solved it. Thank you

**Reply by strawtaco** [reply]
I don't want to use edge and have uninstalled it
But I can't download the driver with just one click on ghelper anymore

**Reply by seerge** [reply]
@strawtaco you don't have to use edge, app just tells system to open link with default browser. You need to set one in windows settings :)

**Reply by strawtaco** [reply]
okok

**Comment by SoembieZ** [comment]
Do you guy have ASUS System Control Interface V3 Version 3.0.1.40 or Below if you have sent it to me please I click the link version 3.0.1.40 but it download new version to me

**Reply by seerge** [reply]
[Version 3.1.40](https://dlcdnets.asus.com/pub/ASUS/GamingNB/Image/Software/SoftwareandUtility/16402/ASUSSystemControlInterfacev3_ASUS_Z_V3.1.40.0_16402.exe)

**Reply by SoembieZ** [reply]
![image](https://github.com/user-attachments/assets/7a2c4f61-b697-429a-a4bc-30557579cab1)  its give me a new version

**Reply by seerge** [reply]
Yes, that's correct one

**Reply by SoembieZ** [reply]
THANK YOU MAN, YOU SAVE MY LIFE.

**Reply by dzuul** [reply]
hey do you guys still have the file? its probably unavailable now because the download link seerge gives doesnt work anymore

**Reply by russiankid133** [reply]
дайте мне тоже пожалуйста ссылку на этот файл
